### ISSUE TITLE
something changed in requirements.txt?

### ISSUE THREAD

**Issue by mobeets** [openaustralia/morph#784]
I'm not sure exactly what's going on, but [my scraper](https://morph.io/mobeets/intriguing-things-scraper) stopped being able to find its database, so any data scraped was immediately forgotten. (This started happening on 2015/4/28, in case that date coincides with some code change?)

My `requirements.txt` file ended up being the culprit, which had previously been working fine as follows:

```
scraperwiki==0.3.11
python-dateutil==2.1
BeautifulSoup==3.2.1
unidecode==0.04.16
```

For whatever reason, I had to change the `scraperwiki==0.3.11` part to `-e git+http://github.com/openaustralia/scraperwiki-python.git@morph_defaults#egg=scraperwiki`, which I found on the python docs [page](https://morph.io/documentation/python) as being the default.

Any ideas why this had to change? Ideally I'd like to use a normal scraperwiki version.

**Comment by henare** [comment]
Hmm, I've no idea why this would've changed. AFAIK it shouldn't have worked before because the scraperwiki library specified in `requirements.txt` would've written data to `scraperwiki.sqlite` and not `data.sqlite`.

Maybe @mlandauer can shed some light?

**Comment by mobeets** [comment]
Yeah, true, but for whatever reason I think it failed in different ways. So before, it couldn't find the database to load the data, so I'd always see things like "1554 records added, 1554 records removed in the database". But at some point around 3 months ago it just started saying "nothing changed in the database".
